access-token-management/src/AccessTokenManagement/HybridDPoPNonceStore.cs

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Duende Software. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See LICENSE in the project root for license information.
+
+using Microsoft.Extensions.Caching.Hybrid;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+namespace Duende.AccessTokenManagement;
+
+/// <summary>
+/// DPoP nonce store using IDistributedCache
+/// </summary>
+public class HybridDPoPNonceStore : IDPoPNonceStore
+{
+    const string CacheKeyPrefix = "DistributedDPoPNonceStore";
+    const char CacheKeySeparator = ':';
+
+    private readonly HybridCache _cache;
+    private readonly ILogger<HybridDPoPNonceStore> _logger;
+
+    /// <summary>
+    /// ctor
+    /// </summary>
+    /// <param name="cache"></param>
+    /// <param name="logger"></param>
+    public HybridDPoPNonceStore(
+        [FromKeyedServices(ServiceProviderKeys.DPoPNonceStore)] HybridCache cache,
```

Comment:
Make a primary ctor?

Reply:
I believe the followup PR does this already, but I did it here anyway